wtpython/display.py

```diff
@@ -41,45 +41,51 @@ def store_results_in_module(
 
 
 class PythonCodeConverter(MarkdownConverter):
-    """Create a custom MarkdownConverter that adds adds python syntax highlighting."""
+    """Custom MarkdownConverter to add python syntax highlighting."""
 
     def convert_pre(self, el: Any, text: str, convert_as_inline: bool) -> str:
-        """Convert the <pre> tag into a code blocked marked with py"""
+        """Add python syntax to all <pre> elements."""
         if not text:
             return ""
-        return "\n```py\n%s\n```\n" % text
+        return """
+        ```python
+        {text}
+        ```
+        """
 
 
 class Sidebar(Widget):
-    """Sidebar with list of questions and possible answers"""
+    """Sidebar widget to display list of questions."""
 
     index: Reactive[int] = Reactive(0)
 
     def set_index(self, index: int) -> None:
-        """Set the current question index"""
+        """Set the current question index."""
         self.index = index
 
     def __init__(
-        self, name: Union[str, None], questions: List[StackOverflowQuestion] = None
+        self,
+        name: Union[str, None],
+        questions: List[StackOverflowQuestion] = None,
     ) -> None:
         if questions is not None:
             self.questions = questions
         super().__init__(name=name)
 
     def get_questions(self) -> str:
-        """Put questions into legible format"""
+        """Format question list."""
         text = ""
         for i, question in enumerate(self.questions):
```

Comment:
Perhaps also use `enumerate(..., start=1)` here, like you did in the previous commit?

Reply:
I fixed all of them, however here there's a balance of comparing to `self.index` and having a 1-indexed presentation format

Reply:
> 
> 
> Other than that seems mostly fine to me. Some well needed changes. Not sure why you reverted the `pre` tag from a multi-line comment? That seemed much nicer to me. If you were having issues with getting it to render you should try wrapping the string in `textwrap.dedent()`.

Yeah, I tried dedent and still had some issues, so I just put it back to how you had it.